packages/server/examples/README.md

```diff
@@ -6,7 +6,7 @@ This directory contains examples of how to use the LlamaIndex Server.
 
 ```bash
 export OPENAI_API_KEY=your_openai_api_key
-npx tsx simple-workflow/calculator.ts
+pnpm run dev
 ```
 
-## Open browser at http://localhost:3000
+## Open browser at http://localhost:4000
```

Comment:
@thucpn isn't the default port 3000 (as in the readme?) or examples should also use 3000 then, otherwise it's confusing